.travis-before-script.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e $DRUPAL_TI_DEBUG
+
+# Ensure the right Drupal version is installed.
+# Note: This function is re-entrant.
+drupal_ti_ensure_drupal
```

Comment:
This script can be completely removed now as all of this is also done by the simpletest runners script.sh